q2cli/tools.py

```diff
@@ -37,14 +37,36 @@ def export_data(path, output_dir):
     result.export_data(output_dir)
 
 
+def show_importable_types(ctx, param, value):
+
```

Comment:
Just a quick minor note: we typically don't include a newline at the beginning of the function body unless there's a docstring associated with it.